api/api/controllers/search_controller.py

```diff
@@ -166,16 +165,16 @@ def _post_process_results(
     return results[:page_size]
 
 
-def get_excluded_providers_query() -> Q | None:
+def get_filtered_providers_query() -> Q | None:
```

Comment:
Nit, but "filtered" reads to me basically the same as "excluded". "enabled", "available", or something along those lines would have a more obviously positive, "these are the providers we serve results from" connotation.